cypress/e2e/brewery/ScenarioMetadataParameters.cy.js

```diff
@@ -54,3 +56,33 @@ describe('scenario metadata parameters', () => {
     apiUtils.waitAliases([scenarioSaveAlias]);
   });
 });
+
+describe('hidden scenario parameters', () => {
+  before(() => {
+    stub.start({
+      GET_DATASETS: true,
+      GET_SCENARIOS: true,
+      GET_WORKSPACES: true,
+      GET_SOLUTIONS: true,
+      UPDATE_SCENARIO: true,
+      LAUNCH_SCENARIO: true,
+    });
+    stub.setScenarios(SCENARIOS);
+    stub.setSolutions(SOLUTIONS);
+  });
+
+  beforeEach(() => {
+    Login.login();
+  });
+
+  after(() => {
+    stub.stop();
+  });
+
+  it('must not show hidden groups of parameters', () => {
+    ScenarioParameters.expandParametersAccordion();
+    // Tabs container must contain only 1 div: the "no parameters" placeholder
+    ScenarioParameters.getParametersTabs().find('div').should('have.length', 1);
```

Comment:
The test just above with scenario metadata parameters already checks that parameters are not visible.
This second test has been added to check that _groups of parameters_ are correctly hidden too. Instead of checking each group and each parameter, we can perform the same verification with only two lines:
- check that the tabs container has only one `div` child
- check that this tab is the "no parameters" placeholder (that is only visible when no parameters are visible)